addons/l10n_it_stock_ddt/report/l10n_it_ddt_report.xml

```diff
@@ -124,7 +124,12 @@
                                         <span t-field="move.product_uom" groups="uom.group_uom"/>
                                     </td>
                                     <td>
-                                        <t t-set="lst_price" t-value="move.product_id.lst_price * move.quantity_done"/>
+                                        <t t-if="move.sale_line_id">
+                                            <t t-set="lst_price" t-value="move.sale_line_id.price_subtotal"/>
+                                        </t>
+                                        <t t-else="">
+                                            <t t-set="lst_price" t-value="move.product_id.lst_price * move.quantity_done"/>
```

Comment:
product_qty -> because it needs to be in the UoM of the lst_price (default UoM of the product)